### PR TITLE
Questions 102 - 108 translated to German

### PR DESCRIPTION
**Pull request by jakeherp**

### CODE REVIEW

File: de-DE/README.md [hunk 1]
```diff
@@ -3225,3 +3225,236 @@ Mit dem `||` Operator geben wir den ersten truthy Operand aus. Wenn alle Werte f
...
+
+```javascript
+const myPromise = () => Promise.resolve('I have resolved!')
+
+function firstFunction() {
+  myPromise().then(res => console.log(res))
+  console.log('second')
+}
+
+function secondFunction() {
```
Comment (lydiahallie) [inline de-DE/README.md]: I made a mistake here in the English translation, this should be `async function secondFunction` 😄

File: de-DE/README.md [hunk 2]
```diff
@@ -3225,3 +3225,236 @@ Mit dem `||` Operator geben wir den ersten truthy Operand aus. Wenn alle Werte f
...
+  if (person1 !== person2) {
+    console.log("Not the same!")
+  } else {
+    console.log("They are the same!")
+  }
+}
+
+const person = { name: "Lydia" }
+
+compareMembers(perosn)
```
Comment (lydiahallie) [inline de-DE/README.md]: `compareMembers(person)`

Comment (jakeherp) [reply de-DE/README.md]: All fixed!

**Comment by jakeherp** [comment]
All fixed!

**Pull request merged**